library/src/main/java/com/otaliastudios/zoom/ZoomApi.kt

```diff
@@ -189,6 +201,16 @@ interface ZoomApi {
      */
     fun setAllowFlingInOverscroll(allow: Boolean)
 
+    /**
+     * Sets the base transformation to be applied to the content.
+     * See [setTransformation].
+     *
+     * @param transformation the transformation type
+     */
+    fun setTransformation(@Transformation transformation: Int) {
+        setTransformation(transformation, TRANSFORMATION_GRAVITY_AUTO)
```

Comment:
I think this can be replaced with a default value on `fun setTransformation(@Transformation transformation: Int, gravity: Int = TRANSFORMATION_GRAVITY_AUTO)` and a `@JvmOverloads` annotation.

Reply:
It's not possible on interfaces unfortunately

Reply:
Ah didn't think about that. Sad :disappointed: